src/reason-parser/reason_pprint_ast.ml

```diff
@@ -5637,9 +5637,9 @@ let printer = object(self:'self)
         | ({txt="JSX"; loc}, PStr []) :: _ ->
           begin match self#simplest_expression x with
             | Some r -> self#formatChildren remaining (r :: processedRev)
-            | None -> self#formatChildren (remaining @ children) processedRev
+            | None -> self#formatChildren (children @ remaining) processedRev
```

Comment:
this has been broken forever 🙈

Reply:
What's this again?

Reply:
Printing of fragments. we wanna print the fragment's children before the fragment's siblings.

Reply:
i.e. depth first vs. breadth first

Reply:
this test:

```reason
<div> <> ...[ [1,2], [3,4] ] </> </div>
```

was getting formatted to

```reason
<div> <> 1 3 2 4 </> </div>
```

i.e. `1 3 2 4` instead of `1 2 3 4` :)

Reply:
Same question as above, I think I expect `<div> <> [1,2] [3,4] </> </div>` here. Am I wrong?

Reply:
yeah, I'm not sure. need someone else to weigh in